src/domain/session/rightpanel/MemberTileViewModel.js

```diff
@@ -47,7 +48,7 @@ export class MemberTileViewModel extends ViewModel {
     }
 
     get detailsUrl() {
-        return `${this.urlCreator.urlUntilSegment("room")}/member/${this._member.userId}`;
+        return `${this.urlCreator.openRoomActionUrl(this._roomId)}/member/${this._member.userId}`;
```

Comment:
you could also get the current room id from `this.navigation.path.get("room").value` so you don't need to inject it ...

Reply:
Oops!🤦‍♂️